__tests__/plugin.test.js

```diff
@@ -16,6 +16,33 @@ describe('babel plugin', () => {
     process.env.REANIMATED_PLUGIN_TESTS = 'jest';
   });
 
+  it('injects its version', () => {
+    const input = `
+      function foo() {
+        '__Reanimated Babel Plugin version injection entry point';
+        var foo = 'bar';
+      }
+    `;
+
+    const { code } = runPlugin(input, {});
+    expect(code).toContain('global._REANIMATED_VERSION_BABEL_PLUGIN = "');
+    expect(code).not.toContain(
+      '__Reanimated Babel Plugin version injection entry point'
+    );
+  });
+
+  it("doesn't bother other Directive Literals", () => {
+    const input = `
+    function foo() {
+      'debugger';
```

Comment:
Can we use something else here, like `'something';`?

Reply:
I wanted to leave 'debugger' as legacy of this development process.